src/components/mobile/ListPage.js

```diff
@@ -0,0 +1,94 @@
+import { useRect } from '@reach/rect'
+import { useEffect, useRef, useState } from 'react'
+import styled from 'styled-components/macro'
+
+import { useMap } from '../../contexts/MapContext'
+import { getLocations } from '../../utils/api'
+import InfiniteList from '../list/InfiniteList'
+// TODO: Ask Jeffrey what the limit for mobile list view should be
+const LIMIT = 30
+
+const ListPageContainer = styled.div`
+  height: 100%;
+`
+
+const LoadingContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+`
+
+const ListPage = () => {
+  const { view } = useMap()
+  const [locations, setLocations] = useState([])
+  const [hasMoreItems, setHasMoreItems] = useState(false)
```

Comment:
Do you need new state for hasMoreItems, or can it be derived from `locations`? E.g. `const hasMoreItems = something locations something`